features/clipboard-events.yml

```diff
@@ -0,0 +1,15 @@
+name: Clipboard events
+description: The `"copy"`, `"cut"`, and `"paste"` events fire on elements when the user starts an action with the clipboard.
+spec: https://w3c.github.io/clipboard-apis/#clipboard-event-interfaces
+group: clipboard
+status:
+  compute_from: api.ClipboardEvent
+compat_features:
+  - api.ClipboardEvent
+  - api.ClipboardEvent.ClipboardEvent
+  - api.ClipboardEvent.clipboardData
+  - api.ClipboardItem.type_image_svg_xml
+  - api.ClipboardItem.type_web
```

Comment:
I'm sorry, I keep noticing things on subsequent review. 🤦‍♂️ 

I think these two stick out a little. The `ClipboardItem` API is much newer than events—don't these actually belong in the `async-clipboard` feature?

Reply:
gah, yes thanks for spotting.